### ISSUE TITLE
Dev Branch: Calls unknown service

### ISSUE THREAD

**Issue by rleongcs** [Blackymas/NSPanel_HA_Blueprint#926]
@edwardtfn I just updated to the latest Dev branch and is seeing the error below. Anything I need to do?

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/61628312/f943eac4-1a73-4b18-8660-a867bab0057d)

**Comment by rleongcs** [comment]
@edwardtfn I updated the TFT and then also "Submit" on this error and then restarted HA. After the reboot, I did not see this error popping up again. I guessed all is good?

**Comment by edwardtfn** [comment]
Yes, all good... That was probably an older version of the Blueprint loaded in the memory.

Please let me know about your experience with this latest changes. I've moved a bunch of things to be processed in ESPHome and hope to have a faster system now, although still missing some things to move.
But anyway, tests are really needed now.

By the way, do you use your panel's relays to control a climate system?

**Comment by rleongcs** [comment]
> Please let me know about your experience with this latest changes.

I noticed with the latest changes, the first 3 icons are not updated properly. The relay icons are not showing when I turned on the relay. The climate icon is also no longer showing what the climate state is. For eg. it was showing the snow icon earlier but now just the thermometer even though it is turned on.

![IMG_1846](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/61628312/e14ef2e5-7a40-4ae9-84df-b18f7629ef9f)
![IMG_1847](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/61628312/d7857468-86a3-44a0-97ba-26b28c35c7c1)



> By the way, do you use your panel's relays to control a climate system?

I do not use the relays to control climate as in my country it is only aircon that we use and the relay amp rating is not enough. I will unfortunately not able to test that portion.
